.github/workflows/master.yml

```diff
@@ -106,10 +106,10 @@ jobs:
       - name: Build and test Kyuubi and Spark with maven w/o linters
         run: |
           if [[ "${{ matrix.java }}" == "8" && "${{ matrix.spark }}" == "3.5" && "${{ matrix.spark-archive }}" == "" ]]; then
-            MVN_OPT="${MVN_OPT} -Pcodecov -Dmaven.javadoc.skip=false -Dmaven.scaladoc.skip=false"
+            MAVEN_ARGS="${MAVEN_ARGS} -Pcodecov -Dmaven.javadoc.skip=false -Dmaven.scaladoc.skip=false"
```

Comment:
in shell, we must use `export MAVEN_ARGS=...` to declare the environment varaiable

Reply:
OK, thanks.